lib/bedrock/index.js

```diff
@@ -27,6 +37,23 @@ module.exports = (data) => {
       }
 
       return itemstates
+    },
+
+    loadHashedRuntimeIds () {
+      const Block = require('prismarine-block')(this)
+      data.blocksByRuntimeId = {}
+      for (let i = 0; i < data.blockStates.length; i++) {
+        const { name, states } = data.blockStates[i]
+        const hash = Block.getHash(name, states)
+        data.blocksByRuntimeId[hash] = { stateId: i, ...data.blocksByStateId[i] }
+      }
+    },
+
+    loadRuntimeIds () {
+      data.blocksByRuntimeId = {}
+      for (let i = 0; i < data.blockStates.length; i++) {
+        data.blocksByRuntimeId[i] = { stateId: i, ...data.blocksByStateId[i] }
+      }
     }
```

Comment:
Things that `return {}`'ed inside this function are apart of the exposed prismarine-registry API. You should put non-API functions outside the returned object